config/consts.ts

```diff
@@ -0,0 +1,14 @@
+export const LINKS = {
+  NEWBIES: '/newbies',
+  MEMBERS: '/members',
+  CONTRIBUTORS: 'https://github.com/Maakaf/maakaf-website/graphs/contributors',
+  MAINTAINERS: '/maintainers', // TODO a
+  PROJECTS: '/projects',
```

Comment:
Do you need the TODO and the comment above?

Reply:
True this comment should not be there